java/org/apache/tomcat/buildutil/translate/Utils.java

```diff
@@ -123,7 +123,7 @@ static String formatValueCommon(String in) {
 
         result = ESCAPE_LEADING_SPACE.matcher(result).replaceAll("\\\\$1");
 
-        result = result.replaceAll("\t", "\\t");
+        result = result.replace("\t", "t");
 
```

Comment:
This is not correct. The literal (unescaped) replacement text should be `\t`

Reply:
The second argument in replaceAll means escape t, not \t.
You can check proofs tests. equals returning true if replace maked to t, not \t

Reply:
It is a bug in the original code. It should be replacing all tabs with the literal string `\t`.

Reply:
For the record, none of the current i18n strings use tabs. I did find some messages using them in early 7.0.x versions, but those messages were never referenced in the Java code.

Reply:
Tx.